src/prognostic_equations/edmfx_closures.jl

```diff
@@ -29,23 +29,23 @@ end
    - nh_presssure_flag - bool flag for if we want/don't want to compute
                          pressure drag
    - ᶠlg - local geometry (needed to compute the norm inside a local function)
-   - ᶠbuoyʲ - updraft buoyancy
-   - ᶠu₃ʲ, ᶠu₃⁰ - Covariant3Vector velocity for updraft and environment
+   - ᶠbuoyʲ - covariant3 or contravariant3 updraft buoyancy
+   - ᶠu3ʲ, ᶠu3⁰ - covariant3 or contravariant3 velocity for updraft and environment
```

Comment:
I use u3 to note that it could be either covariant3 or contravariant3. Let me know if you have naming suggestion :)

Reply:
It would be good to make clear in the docstring which is used when.